clients/ts/signalr/src/HubConnection.ts

```diff
@@ -6,20 +6,21 @@ import { IConnection } from "./IConnection";
 import { CancelInvocationMessage, CompletionMessage, IHubProtocol, InvocationMessage, MessageType, StreamInvocationMessage, StreamItemMessage } from "./IHubProtocol";
 import { ILogger, LogLevel } from "./ILogger";
 import { IStreamResult } from "./Stream";
+import { TextMessageFormat } from "./TextMessageFormat";
 import { Arg, Subject } from "./Utils";
 
 const DEFAULT_TIMEOUT_IN_MS: number = 30 * 1000;
+const DEFAULT_PING_INTERVAL_IN_MS: number = 15 * 1000;
 
 /** Describes the current state of the {@link HubConnection} to the server. */
 export enum HubConnectionState {
-    /** The hub connection is disconnected. */
```

Comment:
How come these JSDoc comments are removed?

Reply:
The comments seemed redundant, but I guess I don't know how/where users are viewing these. Will add the comments back.